source/key-concepts/tax_and_benefit_system.md

```diff
@@ -3,25 +3,36 @@
 ## Definition
 
 The tax and benefit system is the higher-level instance in OpenFisca.
-Its goal is to model the legislation of a country.
 
-Basically a tax and benefit system contains simulation [variables](variables.md) (source code) and [legislation parameters](parameters.md) (data).
+Its goal is to model the legislation of a country (or a more specific domain or jurisdiction).
 
-This instance may host as many versions as there are countries in the world.
+A tax and benefit system contains simulation [variables](variables.md) (source code) and [legislation parameters](parameters.md) (data).
 
-> The OpenFisca core engine is able to simulate any country legislation once it is (partially) represented as source code.
+Instances of OpenFisca are created for specific countries,jurisdiction or domains and they all depend on the OpenFisca core engine.
 
-Therefore you have to instantiate and use the version corresponding to your country of interest.
+> The OpenFisca core engine is able to simulate any country's legislation once it is (partially) represented as source code.
 
-### Application: how to call the Python module
+To instantiate and use OpenFisca requires choosing or creating a version that corresponds to your country, jurisdiction or domain of interest.
 
-The system for France is currently the only one well implemented, so your first action should be:
+The following are examples of OpenFisca country instances:
 
-```py
-# Call module describing the French System
-from openfisca_france import FranceTaxBenefitSystem
+ - OpenFisca Aotearoa
+ - OpenFisca Canada
+ - OpenFisca France
+ - OpenFisca Italia
+ - OpenFisca Japan
+ - OpenFisca Senegal
+ - OpenFisca Tunisia
+ - OpenFisca UK
+ - OpenFisca US
 
-# Initialize the legislation
-tax_benefit_system = FranceTaxBenefitSystem()
+There is also state level instances such as:
+ - OpenFisca New South Wales (Australia)
+ - OpenFisca Missouri (USA)
 
-```
+And City level instances such as:
+ - OpenFisca Paris
+ - OpenFisca City of Vancouver
+
+Each instance of OpenFisca is managed and published as open source digital infrastructure by teams positioned within the countries or jurisdictions they are modelling.
```

Comment:
🤔 Not always true as we have researchers and developers that sometimes create models for other countries:
```suggestion
Each instance of OpenFisca is managed and published as open source digital infrastructure by teams positioned within or linked to the countries or jurisdictions they are modelling.
```